pyedflib/highlevel.py

```diff
@@ -327,19 +327,19 @@ def read_edf(edf_file, ch_nrs=None, ch_names=None, digital=False, verbose=False)
 
     with pyedflib.EdfReader(edf_file) as f:
         # see which channels we want to load
-        available_chs = [ch.upper() for ch in f.getSignalLabels()]
+        available_chs = f.getSignalLabels()
         n_chrs = f.signals_in_file
 
         # find out which number corresponds to which channel
         if ch_names is not None:
             ch_nrs = []
             for ch in ch_names:
-                if not ch.upper() in available_chs:
+                if not ch in available_chs:
```

Comment:
why is this changed? seems unrelated to the PR

Reply:
It was to simplify, but I agree it is unrelated to the PR.